### DISCUSSION TITLE
[Problem] Having both Part Design and Part is confusing

### DISCUSSION THREAD

**Discussion by NXTler** [FreeCAD/FreeCAD#11370] [Discussions that will turn in to issues but need refining]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Hello all,
since I started learning FreeCAD, I have always wondered why there were both the Part and Part Design Workbench. I'm an experienced CAD user, and it seems kind of an odd choice to do. Adding unnecessary complexity to the whole system.
So I asked the Discord community of their opinion on this topic, and here are some of the thoughts I got:
- Having two Workbenches is quite confusing, especially for newcomers.
- There are many cases where the user is tempted to (or even forced to) use both Workbenches at the same time. Which is really blurring the line between them.
- One of the Workbenches has to go. "[...] Either way, they are probably more alike, than they are different."

There have been many approaches proposed on the Discord, here are a few:
- Merging one workbench into the other
- Focus development on one workbench and expand its feature to fully replace the other
- Disable one Workbench by default and present the other as the standard way to modeling
- Develop a completely new workbench with features from both

Since there are so many ideas floating around, this is an open discussion.
Many thanks in advance!

### Full version info

```shell
OS: Manjaro Linux (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.34667 (Git) AppImage
Build type: Release
Branch: main
Hash: af5b8714a8371162addb97dd25c29250fcecf7ad
Python 3.10.12, Qt 5.15.8, Coin 4.0.0, Vtk 9.2.5, OCC 7.6.3
Locale: German/Germany (de_DE)
Installed mods: 
  * PieMenu 1.2.7
  * sheetmetal 0.3.1
  * A2plus 0.4.60n
  * QuickMeasure 2022.10.28
  * Curves 0.6.13
  * CfdOF 1.24.5
  * IconThemes
  * fasteners 0.4.67
```

### Subproject(s) affected?

Other (specify in description)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
Maybe in a first step rename the Part WB to something less confusing for new users. https://forum.freecad.org/viewtopic.php?style=10&t=50256

**Comment by NXTler** [comment]
This would definitely be a step in the right direction.

**Comment by WandererFan** [comment]
The names suck, but there are valid arguments for having both workbenches.  Brand new users deserve something simpler than either.
 
https://forum.freecad.org/viewtopic.php?t=6082
https://forum.freecad.org/viewtopic.php?t=56055
https://forum.freecad.org/viewtopic.php?t=27436

**Reply by M4xTheMan** [reply]
I assume we could guide new users better but both WBs follow different concepts and are therefore needed. Questions should be answered by looking at the wiki https://wiki.freecad.org/Part_and_PartDesign

**Comment by Reqrefusion** [comment]
Before 1.0, all name changes must be reconsidered. Workbench names should not be longer than one word.

**Reply by Reqrefusion** [reply]
Some of @WandererFan  advice is very appropriate.
Part Design - Modeling/Modeller
Part - CSG(Constructive Solid Geometry)(I'm not sure about this, I have reservations about the prevalence of this abbreviation.)
Drawing - Plans/Planer(This is something that definitely needs to change. The semantic affinity with Sketcher is frustrating.)

**Reply by pierreporte** [reply]
One word can be too short. Sheet metal is two words.

I think that the best way would be to merge Part into Part Design, or expand Part Design to match Part capabilities, which is the same.

TechDraw should indeed be renamed Drawing. Name changes can happen in FreeCAD: Path became CAM, Arch will become BIM soon.

**Reply by Reqrefusion** [reply]
Exactly, @pierreporte  if there are going to be name changes, I think now is the time.

Many people oppose combining Part with Part Design because there are differences in philosophy. I think complying with industry standards should come first and be incorporated. But changing the name is a smaller and more reasonable step.

**Comment by kaiwas** [comment]
I'm not against combining workbenches into one. But in my work, bodies, parts, origins are not needed, and I often build “separate” geometry.
I will hope that when the workbenches are united, I will be able to use the approach that is now characteristic of the workbench part.
Those. I would like to be able to work as before (as part WB) even if Part Design WB remains.
